app/models/order_item.rb

```diff
@@ -0,0 +1,9 @@
+class OrderItem < ApplicationRecord
+	belongs_to :order, dependent: :destroy
```

Comment:
check whether dependent option is available for belongs_to association

Reply:
checked